crowdsourcehinter/crowdsourcehinter.py

```diff
@@ -96,10 +96,10 @@ def studio_view(self, context=None):
         frag.add_javascript_url('//cdnjs.cloudflare.com/ajax/libs/mustache.js/0.8.1/mustache.min.js')
         frag.add_css(self.resource_string("static/css/crowdsourcehinter.css"))
         frag.add_javascript(self.resource_string("static/js/src/crowdsourcehinter_studio.js"))
-        print type(self.initial_hints), type(self.target_problem), type(self.generic_hints), str
+        print type(self.initial_hints), type(self.target_problem), type(self.generic_hints)
         frag.initialize_js('CrowdsourceHinterStudio',
-                           {'initial': self.initial_hints,
-                            'generic': self.generic_hints,
+                           {'initial': str(self.initial_hints),
+                            'generic': str(self.generic_hints),
```

Comment:
just a note here that I'm using str here because without it the initial and generic hints didn't show up correctly in studio